include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -171,11 +171,6 @@ SpatialOperatorBase<dim, Number>::initialize_dof_handler_and_constraints()
   }
   print_parameter(pcout, "number of dofs per cell", fe_u->n_dofs_per_cell());
   print_parameter(pcout, "number of dofs (total)", dof_handler_u.n_dofs());
-  if(param.spatial_discretization == SpatialDiscretization::HDIV)
```

Comment:
are the DoFs that are periodic not counted for the L2 case? I think this message would apply in any case, and hence does not really help. Since we are using DG, the periodic BCs only tell the boundary cells, which cells are their neighbors on the other side of the domain, right? So there are never any DoFs reduced ..?